lua/entities/ent_jack_gmod_ezgroundscanner.lua

```diff
@@ -52,10 +53,14 @@ if(SERVER)then
 		self:SetUseType(SIMPLE_USE)
 		self:SetAngles(Angle(-90,0,0))
 		---
+		self.EZupgradable=true
+		self.UpgradeProgress={}
+		self.UpgradeCosts=JMod.CalculateUpgradeCosts(JMod.Config.Craftables["EZ Ground Scanner"] and JMod.Config.Craftables["EZ Ground Scanner"].craftingReqs)
+		---
 		timer.Simple(.01,function()
 			self:GetPhysicsObject():SetMass(200)
 			self:GetPhysicsObject():Wake()
-			self:SetPos(self:GetPos()+Vector(0,0,20))
+			self:SetPos(self:GetPos())
```

Comment:
why was this done

Reply:
Because the model used to be slightly bigger, and as it it when spawned it still drops from about head/chest height.

Reply:
oh ok